components/places/ios/Places/Places.swift

```diff
@@ -910,6 +910,8 @@ public class PlacesWriteConnection: PlacesReadConnection {
                 msg.documentType = documentType.rawValue
             case let .titleObservation(title):
                 msg.title = title
+            case .unknown:
```

Comment:
Looks like a bug that may be easy to miss - e.g. somehow you end up `init`-ing HistoryMetadataObservation wrong, then pass it along here, and never notice that nothing actually happens.

Partially why I don't like being able to represent invalid states! E.g. if `unknown` isn't a thing, you can't pass invalid data into this method at all.